src/hotspot/os/aix/os_aix.cpp

```diff
@@ -448,6 +453,14 @@ static void query_multipage_support() {
       }
     }
 
+    // Can we use mmap with 64K pages? (Should be available with AIX7.3 TL1)
+    {
+      void* p = mmap(NULL, 1000000, PROT_READ | PROT_WRITE, MAP_ANON_64K | MAP_ANONYMOUS | MAP_SHARED, -1, 0);
```

Comment:
Weird size. Why 1mio decimal? Why not 64K?

Reply:
OK, I will use 64*K